library/gui/screen/src/main/java/org/quiltmc/qsl/screen/api/client/Screens.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2016, 2017, 2018, 2019 FabricMC
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.screen.api.client;
+
+import java.util.List;
+import java.util.Objects;
+
+import org.quiltmc.qsl.screen.impl.client.ScreenExtensions;
+import org.quiltmc.qsl.screen.mixin.client.ScreenAccessor;
+
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.font.TextRenderer;
+import net.minecraft.client.gui.screen.Screen;
+import net.minecraft.client.gui.widget.ClickableWidget;
+import net.minecraft.client.render.item.ItemRenderer;
+
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+
+/**
+ * Utility methods related to screens.
+ *
+ * @see ScreenEvents
+ */
+@Environment(EnvType.CLIENT)
+public final class Screens {
```

Comment:
Those probably should be replaced with an injected interface instead.

Reply:
👍 working on it now

Reply:
Would the interface have a different name or just the same name?

Reply:
Well im guessing you mean the same name by replace

Reply:
Made.